make_lvrt_image.sh

```diff
@@ -116,15 +192,14 @@ fi
 
 # Do RPi specific stuff (enable UART, I2C, SPI)
 # https://www.raspberrypi.org/forums/viewtopic.php?t=21632
-# TODO: The raspi-config script needs to access /boot/config.txt and possibly 
-#  other syscalls that are not accessible in the container
 if [ -x $MNT_DIR/usr/bin/raspi-config ]; then
 	echo "Setting Raspberry Pi specific configuration"
+	echo "WARNING: There will be expected errors involving syscalls and device-tree; it's safe to ignore these errors"
```

Comment:
Could you filter the output of the commands producing errors to hide the expected errors instead?

Reply:
Good point, I'll give that a shot.

Reply:
I tried a few things but couldn't come up with an easy way to filter the errors.  I think there's an issue because the commands I want to filter are running in a container.  Doing '2> /dev/null' or 'grep -v errtext' didn't seem to work.

I could probably figure out something if I spent more time on it, but it doesn't seem worth the effort.

Reply:
Ok, no big deal.  Thanks for giving it a host.